code/game/objects/items/weapons/implants/implant_mindslave.dm

```diff
@@ -0,0 +1,78 @@
+/obj/item/weapon/implant/mindslave
+	name = "mindslave implant"
+	desc = "Now YOU too can have your very own mindslave! Pop this implant into anybody and they'll obey any command you give for a limited time."
+	origin_tech = "materials=2;biotech=4;programming=4"
+	activated = 0
+	var/timerid
+	var/slavememory
+
+/obj/item/weapon/implant/mindslave/get_data()
+	var/dat = {"<b>Implant Specifications:</b><BR>
+				<b>Name:</b> Syndicate Mindslave implant MK1<BR>
+				<b>Life:</b> Varies between 15 and 20 minutes.<BR>
+				<b>Important Notes:</b> Personnel injected with this device become loyal to the user and will obey any command given for a limited time.<BR>
+				<HR>
+				<b>Implant Details:</b><BR>
+				<b>Function:</b> Allows user to command anyone implanted to do whatever they want.<BR>
+				<b>Special Features:</b> Person with implant MUST obey any order you give. <BR>
+				<b>Integrity:</b> Implant will last around 15 and 20 minutes."}
+	return dat
+
+/obj/item/weapon/implant/mindslave/implant(mob/target,mob/user)
+	if(target == user)
+		target <<"<span class='notice'>You can't implant yourself!</span>"
+		return 0
+	if(..())
+		target << "<span class='notice'>You feel a surge of loyalty towards [user].</span>"
+		target << "\red You MUST obey any command given to you by your master(that doesn't violate any rules). You are an antag while mindslaved."
```

Comment:
\red isn't really used,you should rather use <span class='danger'> or userdanger if you want a bold version of it, don't forget the ending </span>